apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -354,6 +355,24 @@ const trackErrorNotices = ( content, options ) =>
 		notice_options: JSON.stringify( options ), // Returns undefined if options is undefined.
 	} );
 
+const trackEnableComplementaryArea = ( scope, id ) => {
+	const active = select( 'core/interface' ).getActiveComplementaryArea( scope );
```

Comment:
just a nit on naming, maybe `activeArea` or something similar would be more semantic

Reply:
Yep! Thanks!